Felienne/hedy
# [UI IMPROVEMENT] Improve cheatsheet UI

**Description**
In this PR we improve the UI of the cheatsheets, making a better differentiation between the different commands.

**Fixes**
No related issue (partly discusses and fixes #2904).

**How to test**
Navigate too `/cheatsheet/<level>` and verify that the generated cheatsheet looks as expected and works.

**New view**
![Schermafbeelding 2022-06-16 om 10 19 31](https://user-images.githubusercontent.com/38155424/174026965-ac6d0ca2-ffd8-40fb-9d6b-bfeb451036e5.png)
